build.cake

```diff
@@ -0,0 +1,115 @@
+// Install tools.
+#tool "nuget:https://www.nuget.org/api/v2?package=GitVersion.CommandLine&version=3.6.2"
+
+///////////////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+///////////////////////////////////////////////////////////////////////////////
+
+var target          = Argument<string>("target", "Default");
+var configuration   = Argument<string>("configuration", "Release");
+
+// Load other scripts.
+#load "./build/parameters.cake"
```

Comment:
> Must be excluded through gitignore.

I'm curious about this bit - it looks like `BuildParameters` is defined in this script, so having it ignored would affect the first-start experience after cloning a repository. 

> `error CS0246: The type or namespace name 'BuildParameters' could not be found (are you missing a using directive or an assembly reference?)`

Anyway, I'll see what is inside this file before speculating further :metal: